tests/tests_integration/test_api/test_transformations/test_jobs.py

```diff
@@ -93,6 +94,9 @@ async def other_running_transformation(other_transformation):
 
 class TestTransformationJobsAPI:
     @pytest.mark.asyncio
+    @pytest.mark.skipif(
```

Comment:
Looks like we're adding the skip condition to all tests in this class? Then you can move the decorator to the class level.

Reply:
Done. Ditto for the other file.